src/placeos-models/asset.cr

```diff
@@ -0,0 +1,53 @@
+require "./base/model"
+
+module PlaceOS::Model
+  class Asset < ModelBase
+    include RethinkORM::Timestamps
+
+    table :asset
+
+    attribute name : String, es_subfield: "keyword"
+    attribute category : String = ""
+    attribute description : String = ""
+    attribute size : String = ""
+
+    attribute purchase_date : Int32
+    attribute good_until_date : Int32?
+
+    attribute barcode : Int32
+    attribute brand : String = ""
+
+    attribute purchase_price : Int32
```

Comment:
What's the unit here?

Reply:
Not sure, maybe save as cents and display in dollars in the front end?

Reply:
I think this should still be addressed